eng/Versions.props

```diff
@@ -26,6 +28,8 @@
     <MicrosoftDotNetCoreFxTestingPackageVersion>1.0.0-beta.19105.4</MicrosoftDotNetCoreFxTestingPackageVersion>
     <MicrosoftDotNetBuildTasksConfigurationPackageVersion>1.0.0-beta.19081.1</MicrosoftDotNetBuildTasksConfigurationPackageVersion>
     <MicrosoftDotNetBuildTasksFeedPackageVersion>2.2.0-beta.19081.1</MicrosoftDotNetBuildTasksFeedPackageVersion>
+    <MicrosoftNetCompilersVersion>3.0.0-beta2-final</MicrosoftNetCompilersVersion>
```

Comment:
Does this also need to go into `eng\Version.Details.xml`?

/cc @mmitche

Reply:
Ideally.  Do we know what sha this came from?

Reply:
The files in the nuget package indicate: `3.0.0-beta2-19068-12+231aeb8be8357239499d45c0574e5a9a8c9174f0` (so https://github.com/dotnet/roslyn/commit/231aeb8be8357239499d45c0574e5a9a8c9174f0).

Reply:
Done.